constants/constants.go

```diff
@@ -7,8 +7,11 @@ const (
 	AUTOMATE_PRODUCT      = "automate"
 	HABITAT_PRODUCT       = "habitat"
 	LATEST                = "latest"
+	PLATFORM_SERVICE      = "chef-360"
+	PLATFORM_ERROR        = "chef-360 not available for the trial and opensource "
 )
 
 const (
-	UNMARSHAL_ERR_MSG = "error on unmarshal.\n[ERROR] -"
+	UNMARSHAL_ERR_MSG              = "error on unmarshal.\n[ERROR] -"
+	SuccessResponseFromFilenameLog = "Returning success response from fileName API for "
```

Comment:
pls keep the casing pattern of all constant names in sync.

Reply:
Done